docs/4.0/components/modal.md

```diff
@@ -459,7 +459,11 @@ For modals that simply appear rather than fade in to view, remove the `.fade` cl
 
 ### Dynamic heights
 
-If the height of a modal changes while it is open, you should call `$('#myModal').data('bs.modal').handleUpdate()` or `$('#myModal').modal('handleUpdate')` to readjust the modal's position in case a scrollbar appears.
+If the height of a modal changes while it is open, you should call
+{% highlight js %}$('#myModal').data('bs.modal').handleUpdate(){% endhighlight %}
```

Comment:
Rather than introduce inline highlighting here (which we don't do anywhere else), let's rewrite the paragraph.

> If you change the contents of a modal while it's open, you may need to manually account for changes to the modal's height.
> 
> {% highlight js %}
> // Option 1
> $('#myModal').modal('handleUpdate')
>
> // Option 2
> $('#myModal').data('bs.modal').handleUpdate()
> {% endhighlight %}

Might need better comments than `Option 1/2` though?

Reply:
Any hint as to when to use one option over the other? I admit that this is the part that's currently not explained at all in the docs, and I never grokked it myself (as i never had a need to dynamically alter modal content before)

Reply:
@mdo digging a bit further, it seems the variant with `data(...)` is related to our programmatic API, but it's just a preference (some authors may use this)? as we don't seem to do this consistently anywhere else (we seem to just stick to the documented API methods), I'm actually more inclined to drop this here (as it then avoids having to go off on a tangent on why authors may or may not choose one over the other, which is best kept to the programmatic API docs page anyway).